root/RawLogsScraper.py

```diff
@@ -183,7 +183,7 @@ def scrape(url, serverName, fileBuffer, newestAllowed = 0):
     if "round" in round_name:
         # Fully functional performance logging was merged on the 10th of November 2020, this prevents overshooting when taking the initial copy of the logs 
         current_id = round_name.split("-")[1] 
-        if int(current_id) <= target_round:  
+        if len(current_id.split(".")) > 2 or int(current_id) <= target_round:  
```

Comment:
int('xx.yy.zz') runtimes without this

Reply:
fixed with an isnumeric(), s dumb, I hate rounds with no DB